tools/api-markdown-documenter/api-report/api-markdown-documenter.api.md

```diff
@@ -501,50 +501,50 @@ function renderHeritageTypes(apiItem: ApiItem, config: Required<MarkdownDocument
 
 declare namespace RenderingHelpers {
     export {
-        renderSignature,
-        renderSeeAlso,
-        renderHeritageTypes,
-        renderTypeParameters,
-        renderExcerptWithHyperlinks,
-        renderBreadcrumb,
-        renderHeadingForApiItem,
-        renderHeading,
+        ChildSectionProperties,
+        DocExampleProperties,
         renderBetaAlert,
-        renderSummarySection,
-        renderRemarksSection,
-        renderThrowsSection,
+        renderBreadcrumb,
+        renderChildDetailsSection,
+        renderChildrenUnderHeading,
         renderDeprecationNoticeSection,
-        renderExamplesSection,
         renderExampleSection,
+        renderExamplesSection,
+        renderExcerptWithHyperlinks,
+        renderHeading,
+        renderHeadingForApiItem,
+        renderHeritageTypes,
+        renderLink,
         renderParametersSection,
+        renderRemarksSection,
         renderReturnsSection,
-        renderChildDetailsSection,
-        renderChildrenUnderHeading,
-        renderLink,
-        DocExampleProperties,
-        ChildSectionProperties,
-        renderMemberTables,
-        renderTableWithHeading,
-        renderSummaryTable,
-        renderDefaultSummaryTable,
-        renderParametersSummaryTable,
-        renderFunctionLikeSummaryTable,
-        renderPropertiesTable,
-        renderPackagesTable,
+        renderSeeAlso,
+        renderSignature,
+        renderSummarySection,
+        renderThrowsSection,
+        renderTypeParameters,
+        MemberTableProperties,
```

Comment:
Yes it's autogenerated. This autogeneration is also doing quetionable things on this PR -- https://github.com/microsoft/FluidFramework/pull/12424#discussion_r993801849!!

Reply:
Never mind, the other PR is something else - it's just that it's getting populated for the first time with everything. The reason the M is sneaking in here is because of it's position in the ts file: https://github.com/microsoft/FluidFramework/pull/12463/files/8cfdd2e22aa0602bd7d709f4a6733f7da288b159#r995921032

If you look at this original file it doesn't maintain alphabetization on it's own.